src/runtime/interop.cs

```diff
@@ -320,34 +360,34 @@ public static void FreeModuleDef(IntPtr ptr)
     /// </summary>
     internal class TypeFlags
     {
-        public static int HeapType = (1 << 9);
-        public static int BaseType = (1 << 10);
-        public static int Ready = (1 << 12);
-        public static int Readying = (1 << 13);
-        public static int HaveGC = (1 << 14);
+        public const int HeapType = (1 << 9);
+        public const int BaseType = (1 << 10);
```

Comment:
This really should be an enum

Reply:
@lostmsu done.  It did require more changes elsewhere but nothing too drastic.  Good thing TypeFlags is internal or we'd have to document it as a breaking change.